superset/viz.py

```diff
@@ -509,7 +509,7 @@ def query_obj(self):
                 'Choose either fields to [Group By] and [Metrics] or '
                 '[Columns], not both'))
 
-        sort_by = fd.get('timeseries_limit_metric') or []
+        sort_by = fd.get('timeseries_limit_metric') or None
```

Comment:
You can remove the `or None` here, if get fails sort_by will be None

Reply:
fixed. thanks!